autogpt_platform/backend/backend/data/notifications.py

```diff
@@ -39,7 +39,7 @@ class AgentRunData(BaseNotificationData):
     execution_time: float
     node_count: int = Field(..., description="Number of nodes executed")
     graph_id: str
-    outputs: dict[str, Any] = Field(..., description="Outputs of the agent")
+    outputs: list[dict[str, Any]] = Field(..., description="Outputs of the agent")
```

Comment:
You can use `CompletedBlockOutput` type

Reply:
This does not work so good because that type is not hashable which means dict comprehension does not work